src/utilities/fastify/handler/setErrorHandler.ts

```diff
@@ -0,0 +1,20 @@
+import type { FastifyInstance } from 'fastify';
+
+import { HTTP_STATUS_CODES } from '../../http.js';
+
+export default function setErrorHandler(fastify: FastifyInstance): void {
+  const internalServerError = 'Internal server error';
+  fastify.setErrorHandler(async (error, _request, reply) => {
+    if (
+      error.statusCode !== undefined &&
+      error.statusCode < HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR
+    ) {
+      fastify.log.info({ err: error }, 'Client error');
+      await reply.send(error);
+      return;
+    }
+
+    fastify.log.error({ err: error }, internalServerError);
```

Comment:
I don't think the log message should be "Internet server error" here: We don't know if it's a `500`. It could be any other `5XX` error.

I think we should use something like `Request processing failed due to server error`.

It does, however, make sense to return a response body of `Internal server error` to the client. So let's keep that part -- without the variable `internalServerError` as it won't be necessary anymore.